ppb/sprites.py

```diff
@@ -185,49 +212,46 @@ def center(self) -> Vector:
 
     @center.setter
     def center(self, value: Sequence[float]):
```

Comment:
```suggestion
    def center(self, value: ppb_vector.vector2.VectorLike):
```

This change should likely wait until we cut a release of `ppb-vector` though.